Support for "application/zip" accepted mediatype

Currently suite is not supporting "application/zip" as the result type. The reason is the quite old version of the teamengine-spi dependency. 

Would it be possible to bump up the version? The reason I would like to have this is because it would generate the result as html which is more human readable that the earl or xml report.

@bpross-52n 
When I am running the jar by using https://github.com/opengeospatial/ets-wfs20/blob/dd77f2b3fd1b52dd24ff5f4d5d7cf87f908a00d3/src/main/resources/test-run-props.xml and following command
```
stenger@elsbeere:~/git/ets-wfs20/target$ java -jar ets-wfs20-1.37-SNAPSHOT-aio.jar test-run-props.xml
```
errors are returned and no HTML report is generated:
```
[TestNG] Reporter com.occamlab.te.spi.executors.testng.EarlReporter@c8e4bb0 failed
java.lang.NullPointerException
	at com.occamlab.te.spi.executors.testng.EarlReporter.isBasicConformanceClass(EarlReporter.java:451)
	at com.occamlab.te.spi.executors.testng.EarlReporter.addTestRequirements(EarlReporter.java:240)
	at com.occamlab.te.spi.executors.testng.EarlReporter.generateReport(EarlReporter.java:103)
	at org.testng.TestNG.generateReports(TestNG.java:1135)
	at org.testng.TestNG.run(TestNG.java:1081)
	at com.occamlab.te.spi.executors.testng.TestNGExecutor.execute(TestNGExecutor.java:137)
	at org.opengis.cite.iso19142.TestNGController.doTestRun(TestNGController.java:142)
	at org.opengis.cite.iso19142.TestNGController.main(TestNGController.java:82)
Error 
  java.io.FileNotFoundException:
  /home/stenger/testng/e9b58970-67d6-4b70-970d-281b649ec403/earl-results.rdf (Datei oder
  Verzeichnis nicht gefunden)
Jan 20, 2022 9:11:13 PM com.occamlab.te.spi.util.HtmlReport earlHtmlReport
SCHWERWIEGEND: Transformation of EARL to HTML failed.
net.sf.saxon.trans.XPathException: java.io.FileNotFoundException: /home/stenger/testng/e9b58970-67d6-4b70-970d-281b649ec403/earl-results.rdf (Datei oder Verzeichnis nicht gefunden)
	at net.sf.saxon.event.Sender.sendSAXSource(Sender.java:384)
	at net.sf.saxon.event.Sender.send(Sender.java:185)
	at net.sf.saxon.event.Sender.send(Sender.java:49)
	at net.sf.saxon.Controller.transform(Controller.java:1550)
	at com.occamlab.te.spi.util.HtmlReport.earlHtmlReport(HtmlReport.java:102)
	at com.occamlab.te.spi.util.HtmlReport.getHtmlResultZip(HtmlReport.java:62)
	at com.occamlab.te.spi.executors.testng.TestNGExecutor.getResultsFile(TestNGExecutor.java:172)
	at com.occamlab.te.spi.executors.testng.TestNGExecutor.execute(TestNGExecutor.java:140)
	at org.opengis.cite.iso19142.TestNGController.doTestRun(TestNGController.java:142)
	at org.opengis.cite.iso19142.TestNGController.main(TestNGController.java:82)
Caused by: java.io.FileNotFoundException: /home/stenger/testng/e9b58970-67d6-4b70-970d-281b649ec403/earl-results.rdf (Datei oder Verzeichnis nicht gefunden)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:90)
	at sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:188)
	at org.apache.xerces.impl.XMLEntityManager.setupCurrentEntity(Unknown Source)
	at org.apache.xerces.impl.XMLVersionDetector.determineDocVersion(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at net.sf.saxon.event.Sender.sendSAXSource(Sender.java:371)
	... 9 more
---------
java.io.FileNotFoundException: /home/stenger/testng/e9b58970-67d6-4b70-970d-281b649ec403/earl-results.rdf (Datei oder Verzeichnis nicht gefunden)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:90)
	at sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:188)
	at org.apache.xerces.impl.XMLEntityManager.setupCurrentEntity(Unknown Source)
	at org.apache.xerces.impl.XMLVersionDetector.determineDocVersion(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at net.sf.saxon.event.Sender.sendSAXSource(Sender.java:371)
	at net.sf.saxon.event.Sender.send(Sender.java:185)
	at net.sf.saxon.event.Sender.send(Sender.java:49)
	at net.sf.saxon.Controller.transform(Controller.java:1550)
	at com.occamlab.te.spi.util.HtmlReport.earlHtmlReport(HtmlReport.java:102)
	at com.occamlab.te.spi.util.HtmlReport.getHtmlResultZip(HtmlReport.java:62)
	at com.occamlab.te.spi.executors.testng.TestNGExecutor.getResultsFile(TestNGExecutor.java:172)
	at com.occamlab.te.spi.executors.testng.TestNGExecutor.execute(TestNGExecutor.java:140)
	at org.opengis.cite.iso19142.TestNGController.doTestRun(TestNGController.java:142)
	at org.opengis.cite.iso19142.TestNGController.main(TestNGController.java:82)

Exception in thread "main" java.lang.RuntimeException: net.sf.saxon.trans.XPathException: java.io.FileNotFoundException: /home/stenger/testng/e9b58970-67d6-4b70-970d-281b649ec403/earl-results.rdf (Datei oder Verzeichnis nicht gefunden)
	at com.occamlab.te.spi.util.HtmlReport.earlHtmlReport(HtmlReport.java:108)
	at com.occamlab.te.spi.util.HtmlReport.getHtmlResultZip(HtmlReport.java:62)
	at com.occamlab.te.spi.executors.testng.TestNGExecutor.getResultsFile(TestNGExecutor.java:172)
	at com.occamlab.te.spi.executors.testng.TestNGExecutor.execute(TestNGExecutor.java:140)
	at org.opengis.cite.iso19142.TestNGController.doTestRun(TestNGController.java:142)
	at org.opengis.cite.iso19142.TestNGController.main(TestNGController.java:82)
Caused by: net.sf.saxon.trans.XPathException: java.io.FileNotFoundException: /home/stenger/testng/e9b58970-67d6-4b70-970d-281b649ec403/earl-results.rdf (Datei oder Verzeichnis nicht gefunden)
	at net.sf.saxon.event.Sender.sendSAXSource(Sender.java:384)
	at net.sf.saxon.event.Sender.send(Sender.java:185)
	at net.sf.saxon.event.Sender.send(Sender.java:49)
	at net.sf.saxon.Controller.transform(Controller.java:1550)
	at com.occamlab.te.spi.util.HtmlReport.earlHtmlReport(HtmlReport.java:102)
	... 5 more
Caused by: java.io.FileNotFoundException: /home/stenger/testng/e9b58970-67d6-4b70-970d-281b649ec403/earl-results.rdf (Datei oder Verzeichnis nicht gefunden)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:90)
	at sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:188)
	at org.apache.xerces.impl.XMLEntityManager.setupCurrentEntity(Unknown Source)
	at org.apache.xerces.impl.XMLVersionDetector.determineDocVersion(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at net.sf.saxon.event.Sender.sendSAXSource(Sender.java:371)
	... 9 more
```

Can you please describe the workflow how you executed the jar?

--- a/pom.xml
+++ b/pom.xml
@@ -69,7 +69,7 @@
     <dependency>
       <groupId>org.opengis.cite.teamengine</groupId>
       <artifactId>teamengine-spi</artifactId>
-      <version>5.4</version>
+      <version>5.4.1</version>
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>

--- a/pom.xml
+++ b/pom.xml
@@ -69,6 +69,7 @@
     <dependency>
       <groupId>org.opengis.cite.teamengine</groupId>
       <artifactId>teamengine-spi</artifactId>
+      <version>5.4</version>
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>

--- a/src/main/config/test-run-props.xml
+++ b/src/main/config/test-run-props.xml
@@ -3,5 +3,5 @@
 <properties version="1.0">
     <comment>Sample test run arguments (ets-wfs20)</comment>
     <entry key="wfs">wfs2-capabilities.xml</entry>
-    <entry key="acceptMediaType">application/zip</entry>    
+    <entry key="acceptMediaType">application/zip</entry>
 </properties>

--- a/src/main/config/test-run-props.xml
+++ b/src/main/config/test-run-props.xml
@@ -3,4 +3,5 @@
 <properties version="1.0">
     <comment>Sample test run arguments (ets-wfs20)</comment>
     <entry key="wfs">wfs2-capabilities.xml</entry>
+    <entry key="acceptMediaType">application/zip</entry>    
 </properties>

--- a/src/main/resources/test-run-props.xml
+++ b/src/main/resources/test-run-props.xml
@@ -2,5 +2,6 @@
 <!DOCTYPE properties SYSTEM "http://java.sun.com/dtd/properties.dtd">
 <properties version="1.0">
     <comment>WFS 2.0 reference implementation</comment>
-    <entry key="wfs">http://cite.lat-lon.de/deegree-webservices-3.3.6/services/wfs200?request=GetCapabilities&amp;service=WFS&amp;version=2.0.0</entry>
+    <entry key="wfs">http://cite.deegree.org/deegree-webservices-3.4.16/services/wfs200?request=GetCapabilities&amp;service=WFS&amp;version=2.0.0</entry>
+    <entry key="acceptMediaType">application/zip</entry>
 </properties>

